app/src/main/resources/application.yml

```diff
@@ -95,6 +95,23 @@ spring:
   http:
     encoding:
       force: true
+  security:
+    oauth2:
+      client:
+        provider:
+          mas:
+            uri: ${mas-api-auth-token-uri:https://viccs-api-dev.ibm-intelligent-automation.com/pca/api/dev/token}
+        registration:
+          mas:
```

Comment:
Do you need to add the credentials in the secret project to put them in the environment variables?

Reply:
Removed the credentials